tests/sentry/ingest/test_span_desc_clusterer.py

```diff
@@ -83,51 +81,6 @@ def test_distribution():
     assert freshness > 800, freshness
 
 
-@mock.patch("sentry.ingest.transaction_clusterer.datasource.redis._record_sample")
-@django_db_all
-@pytest.mark.parametrize(
-    "description, description_scrubbed, op, feat_flag_enabled, expected",
-    [
-        ("", "", "http.client", True, 0),
-        ("", "GET /a/b/c", "something.else", True, 0),
-        ("", "GET /a/b/c", "http.client", True, 1),
-        ("GET /a/b/c", "", "something.else", True, 0),
-        ("GET /a/b/c", "", "http.client", True, 1),
-        ("GET /a/b/c", "GET /a/*/c", "something.else", True, 0),
-        ("GET /a/b/c", "GET /a/*/c", "http.client", True, 1),
-        ("GET /a/b/c", "GET /a/*/c", "http.client", False, 0),
-    ],
-)
-def test_record_span(
-    mocked_record,
-    default_organization,
-    description,
-    description_scrubbed,
-    op,
-    feat_flag_enabled,
-    expected,
-):
-    with Feature(
-        {
-            "projects:span-metrics-extraction": feat_flag_enabled,
-        }
-    ):
-        project = Project(id=111, name="project", organization_id=default_organization.id)
-        record_span_descriptions(
-            project,
-            {
-                "spans": [
-                    {
-                        "description": description,
-                        "op": op,
-                        "data": {"description.scrubbed": description_scrubbed},
-                    }
-                ]
-            },
-        )
-        assert len(mocked_record.mock_calls) == expected
-
-
```

Comment:
We only look at the scrubbed description for resource spans, so this test case seemed redundant (there is already a `test_record_span_desc_url`).